src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -208,6 +208,27 @@ public static Category parseCategory (String category) throws ParseException {
         return new Category(trimmedCategory);
     }
 
+    /**
+     * Parses a {@code String highOrLow}
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @param highOrLow to sort by ascending or descending order.
+     * @return the String
+     */
+    public static String parsehighOrLow(String highOrLow) throws ParseException {
```

Comment:
Suggestion to change the method name to a more meaningful one such as `parseFixedExpenseSortCommandKeyword()` or `parseFixedExpenseSortCommandMethod()` any other meaningful name. Same for the parameter, examples are `sortBy`, `sortCommandKeyword` or `sortCommandMethod`, etc.